core/src/main/java/org/infinispan/interceptors/impl/DistCacheWriterInterceptor.java

```diff
@@ -88,7 +91,9 @@ public Object visitPutMapCommand(InvocationContext ctx, PutMapCommand command) t
          PutMapCommand putMapCommand = (PutMapCommand) rCommand;
          LocalizedCacheTopology cacheTopology = dm.getCacheTopology();
          Map<Object, Object> map = putMapCommand.getMap();
-         int count = 0;
+
```

Comment:
What's the benefit of this as the lambda is not stateless? The action taken by the lambda depends on the `InvocationContext` and `PutMapCommand` (flags) that are passed to visitPutMapCommand.